doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,185 @@ To send bare Lua code for execution, use ``Eval``:
 .. code-block:: go
 
     resp, err = connection.Eval("return 4 + 5", []interface{}{})
+
+.. _getting_started-go-comparison:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Feature comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two more connectors from open-source community available:
+`viciious/go-tarantool <https://github.com/viciious/go-tarantool>`_ and
+`FZambia/tarantool <https://github.com/FZambia/tarantool>`_. 
+
+Last update: December 2021
```

Comment:
Seems we can update date here.

Reply:
Updated both here and in tarantool/go-tarantool last update